components/homme/src/share/namelist_mod.F90

```diff
@@ -977,6 +994,11 @@ subroutine readnl(par)
     dx_ref = 1.0D0/ne_x
     dy_ref = 1.0D0/ne_y
 
+    if (Lx==0.0 .or. Ly==0.0) then
```

Comment:
Should it be something like abs(Lx) <= tolerance? 
Overall, inputs for planar in homme need some cleaning, this is out of scope for this PR.